content/_includes/page/list.njk

```diff
@@ -34,5 +34,22 @@
       'Events',
       collections.all | getCollectionItem(page) | pageEvents
     ) }}
+
+    {% if is_entry %}
+      <section data-list="tag" data-typeset>
+      {{ post.recent_posts(collections['Talk'], collections, banner_title="Talks", page_url=page.url)}}
```

Comment:
I notice that this shows "Talk" on e.g. workshop pages as well, since it's really for any page that has events. Are we okay with that, or is there a different term we want to use?

Reply:
@mirisuzanne I'm curious what you think about this?

Reply:
I don't feel very strongly about it. I think there's another story for better handling of talks vs events - this might be good to note as part of that story?